ports/geckolib/properties.mako.rs

```diff
@@ -610,6 +610,23 @@ fn static_assert() {
 
 </%self:impl_trait>
 
+<%self:impl_trait style_struct_name="Color"
+                  skip_longhands="color"
+                  skip_additionals="*">
```

Comment:
If there aren't any additional methods, I don't think you need the skip_additionals="*".